pymongo/topology.py

```diff
@@ -128,15 +145,17 @@ def __init__(self, topology_settings):
         self._opened = False
         self._closed = False
         self._lock = _create_lock()
+        assert self._settings.condition_class is not None
```

Comment:
condition_class should be changed to not optional.

Reply:
done